plugins/SEO/Metric/DomainAge.php

```diff
@@ -74,16 +74,12 @@ public function getMetrics($domain)
      */
     private function getAgeArchiveOrg($domain)
     {
-        $data = $this->getUrl('https://wayback.archive.org/web/*/' . urlencode($domain));
-        preg_match('#<a href=\"([^>]*)' . preg_quote($domain) . '/\">([^<]*)<\/a>#', $data, $p);
-        if (!empty($p[2])) {
-            $value = strtotime($p[2]);
-            if ($value === false) {
-                return 0;
-            }
-            return $value;
+        $response = $this->getUrl('https://archive.org/wayback/available?timestamp=19900101&url=' . urlencode($domain));
+        $data = json_decode($response, true);
+        if (empty($data["archived_snapshots"])) {
```

Comment:
to be safe... should this be `if (empty($data["archived_snapshots"]["closest"]["timestamp"]))` ?

Reply:
Archive.org is currently down for maintenance, so I can't test it.
I thought that would throw an error if `$data["archived_snapshots"]["closest"]` doesn't exist, but it seems like the PHP empty function takes care of that.